### PR TITLE
Expand example content for extended markdown usage

### PR DESCRIPTION
**Pull request by ksubburayalu**
Expand example content for extended markdown usage for clarity.

### Review

@reviewer

### CODE REVIEW

File: app/enterprise/2.3.x/developer-portal/theme-customization/markdown-extended.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ improves rendering significantly, especially for tables.
 
 ## Specify the markdown rendering module in a document
 
-1. Create a markdown file for your Dev Portal documentation.
+1. Create a markdown file for your Dev Portal documentation. The markdown file, for example `markdown-example.md` needs to be added under `content` folder. Add your markdown content, refer [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/) to start with. The example below has few content for your references.
```
Comment (tlblessing) [inline app/enterprise/2.3.x/developer-portal/theme-customization/markdown-extended.md:22]:
```suggestion
1. Create a markdown file for your Dev Portal documentation. The markdown file, for example `markdown-example.md`, needs to be added under the `content` folder. Add your markdown content. Refer to the [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/). The example below shows some basic markdown syntax.
```

**Review by tlblessing — CHANGES_REQUESTED** [review]
Thanks for the PR @ksubburayalu. Some suggested edits for correct grammar.

**Comment by Bradamant3** [comment]
Hey folks, please hold off on approving/merging. The actual markdown example changes need fixing up too, I'll get to them after I'm done with bintray migration for Mesh. Thanks for the English fixes, @tlblessing !

**Comment by Bradamant3** [comment]
@tlblessing I edited the English a little differently bc I needed to pull the branch down to fix up the markdown, too. PTAL!

**Comment by ksubburayalu** [comment]
Yes, this was done based on a customer issue. Refer the ticket 21753 that I have adde as part of the branch name as well. 
Thanks

**Comment by ksubburayalu** [comment]
Docs team, can this be merged? Any other open items to be addressed? Thanks

**Review by Bradamant3 — APPROVED** [review]
LGTM but I pushed a commit so can I do this?

**Pull request merged**